serialization.go

```diff
@@ -88,6 +122,28 @@ func SendJSON(w http.ResponseWriter, ans any) {
 	}
 }
 
+// SendError sends an error.
+// Declared as a variable to be able to override it for clients that need to customize serialization.
+var SendError = func(w http.ResponseWriter, r *http.Request, err error) {
+	accept := parseAcceptHeader(r.Header.Get("Accept"), nil)
+	if accept == "" {
```

Comment:
I believe this is redundant? You'll just fall through and the default condition if it's empty

Reply:
Yes exact, Removing this :)